docs/getting-started-guides/kubeadm.md

```diff
@@ -160,15 +160,14 @@ Before you can deploy applications to it, you need to install a pod network.
 You must install a pod network add-on so that your pods can communicate with each other when they are on different hosts.
 **It is necessary to do this before you try to deploy any applications to your cluster.**
 
-Several projects provide Kubernetes pod networks.
-You can see a complete list of available network add-ons on the [add-ons page](/docs/admin/addons/).
+Several projects are available that enable Kubernetes pod networks including Calico, Romana and Weave. Some may also support [Network Policy](/docs/user-guide/networkpolicies/). See the [add-ons page](/docs/admin/addons/) for a complete list of available network add-ons.
```

Comment:
I agree. Finding/updating all the references to network providers in the current docs is already madness.  Not just network providers, either. Cloud providers, OSs, confi mgt, etc. Here is an example of a page with callouts gone wild. http://kubernetes.io/docs/getting-started-guides/

But there are/were places where there is just one referenced, which is inaccurate and the simplest (and more polite) path forward is not to delete anything and just add more. Happy to follow whatever style/use guide that can be applied.